cogip/widgets/gameview.py

```diff
@@ -270,6 +273,9 @@ def asset_ready(self):
             self.robot_manual = RobotManualEntity(self.root_entity, self.container)
             self.robot_manual.enable_controller.connect(self.camera_controller.setEnabled)
             self.new_move_delta.connect(self.robot_manual.new_move_delta)
+
+            print(f"Load time of assets: {timeit.default_timer() - self.start_time:0.3f}s")
```

Comment:
I like this kind of debug information. Could it be (in the future)  used widely to monitor times in all components (dashboard, monitor, etc...) ?

Reply:
yes the idea is to keep that in mind. thx